File: src/main/java/org/example/eshop/controller/AccountViewController.java
package org.example.eshop.controller;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class AccountViewController {

    @GetMapping("/login")
    public String loginPage() {
        return "login";
    }

    @GetMapping("/account")
    public String accountPage(Model model) {
        return "account";
    }
}

File: src/main/kotlin/org/example/eshop/controller/AccountViewController.kt
package org.example.eshop.controller

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class AccountViewController {

    @GetMapping("/login")
    fun loginPage(): String {
        return "login"
    }

    @GetMapping("/account")
    fun accountPage(model: Model): String {
        return "account"
    }
}
